enterprise/api/v1/views.py

```diff
@@ -197,7 +197,7 @@ class EnterpriseCourseEnrollmentViewSet(EnterpriseReadWriteModelViewSet):
     API views for the ``enterprise-course-enrollment`` API endpoint.
     """
 
-    queryset = models.EnterpriseCourseEnrollment.objects.all()
+    queryset = models.EnterpriseCourseEnrollment.objects.order_by("created").all()
```

Comment:
Can we set ordering on model level: https://docs.djangoproject.com/en/2.0/ref/models/options/#django.db.models.Options.ordering ?
nit: use single quotes

Reply:
We can use the ordering at the module level, but then it will affect everywhere the models are being used. I am not sure if that would be a good idea.

Reply:
Please try to use model level ordering and see if any test fails with the change. If no issues are detected I would suggest that we use model level ordering.